### ISSUE TITLE
Unselecting the "hover" option doesn't do anything

### ISSUE THREAD

**Issue by Thomas-N** [ubuntu-mate/mate-menu#44]
Deselecting the "hover" options under "prefererences" -> "applications" doesn't stop "hover" behaviour in the menu. I think I remember this working properly a long time ago, but it doesn't work now.

Using the advanced Mate menu from the repositories, both in Ubuntu Mate 18.04 and in Arch Linux.

**Comment by procedural** [comment]
Yeah, same issue on 18.04. Fortunately, I'm a programmer, so let's investigate...

**Comment by procedural** [comment]
https://github.com/ubuntu-mate/mate-menu/blob/18.04.3/lib/mate-menu-config.py#L218
https://github.com/ubuntu-mate/mate-menu/blob/18.04.3/mate_menu/plugins/applications.py#L434

**Comment by procedural** [comment]
/usr/lib/mate-menu

**Comment by procedural** [comment]
/usr/share/mate-menu/plugins

**Comment by procedural** [comment]
https://github.com/ubuntu-mate/mate-menu/blob/18.04.3/mate_menu/plugins/applications.py#L434
> StartFilter

**Comment by procedural** [comment]
/usr/lib/python2.7/dist-packages/mate_menu/plugins

**Comment by procedural** [comment]
> /usr/lib/python2.7/dist-packages/mate_menu/plugins

Yea, this one...

**Comment by procedural** [comment]
Ok...

**Comment by procedural** [comment]
So, open a terminal and run:
```bash
sudo cp /usr/lib/python2.7/dist-packages/mate_menu/plugins/applications.py /usr/lib/python2.7/dist-packages/mate_menu/plugins/applications.py.old
sudo pluma /usr/lib/python2.7/dist-packages/mate_menu/plugins/applications.py
```
In the opened file, replace these lines:
https://github.com/ubuntu-mate/mate-menu/blob/18.04.3/mate_menu/plugins/applications.py#L1443-L1452
With this:
```py
                        #if self.categories_mouse_over:
                        #    startId = item["button"].connect( "enter", self.StartFilter, item["filter"] )
                        #    stopId = item["button"].connect( "leave", self.StopFilter )
                        #    item["button"].mouseOverHandlerIds = ( startId, stopId )
                        #else:
                        item["button"].mouseOverHandlerIds = None

                        item["button"].connect( "clicked", self.FilterAndClear, item["filter"] )
                        #item["button"].connect( "focus-in-event", self.categoryBtnFocus, item["filter"] )
                        item["button"].show()
```

And it should work.

**Comment by vkareh** [comment]
@procedural thanks for debugging this, I've submitted a pull request that fixes the behaviour of the Hover option: https://github.com/ubuntu-mate/mate-menu/pull/84

**Comment by procedural** [comment]
@vkareh thanks!

**Comment by HubKing** [comment]
Is this feature coming? I have been using this menu for years, and when I type some letters and try to click one of the search result with the mouse, the items displayed usually are gone because the mouse cursor often passed above a category unintentionally. I saw "hover" in the options, but unchecking it had no effect, so I came hear for asking a feature to disable hovering over categories.
